### ISSUE TITLE
[feature request] Filter all local branches

### ISSUE THREAD

**Issue by narf** [ForkIssues/TrackerWin#1850]
For junior git users it could be very helpful to have a way to display only local branches and their remote.
A simple way to do it could be to add the same hover toggle buttons that there is on individual branches on the "Branches" collapsible group

![image](https://user-images.githubusercontent.com/458263/227877111-573ed5ef-ebb2-4afb-a208-7904b8ee48ab.png)

**Comment by DanPristupov** [comment]
Junior git users will not find and click that button in the first case.

As an alternative, there is 'Filter active branch' button on the status control on the toolbar (Ctrl+Shift+A).

**Comment by narf** [comment]
Well, there is often a senior behind a junior that give him hints and tricks for a easy start. We have lot of work in the team and the git tree is a christmas tree. And our junior often works on multiple branches when peer reviews lags behind (it happens quite often). So A way to focus only on "my relevant work" would be very nice.

But the ctrl-shift-A trick is a good workaround for focus on a single branch, I'm just afraid the juniors will just forget all other local branches they have ^^'

**Comment by DanPristupov** [comment]
> So A way to focus only on "my relevant work" would be very nice.

Add prefix to your local branches and you'll be able to filter them at once. For example `vincent/feature1`, then set filter on both the `vincent` folder and `master`. That is exactly what you want.


P.S. 
> We have lot of work in the team and the git tree is a christmas tree. 

Use 'Lean branching' and 'sync' (https://github.com/fork-dev/TrackerWin/issues/1432)

**Comment by narf** [comment]
> Add prefix to your local branches and you'll be able to filter them at once. For example vincent/feature1, then set filter on both the vincent folder and master. That is exactly what you want.

Thanks for the tip, that's a good idea. Too bad we automated that with our task name, we could probably change the automation but we would have to deal with some edge case. I'll see if I can tweak it in a way or another.

> Use 'Lean branching' and 'sync' (https://github.com/fork-dev/TrackerWin/issues/1432)

As a matter of fact, we already do that and our history is clean, but our many branches being worked on are not:
![image](https://user-images.githubusercontent.com/458263/227900455-22a30ec5-dd3b-4ed2-87ef-15922fcf872d.png)

**Comment by Otiel** [comment]
I second the original request, but as a senior developer. 😄

I'd like to be able to filter all local branches because in my use case they are often related and I'm working on them simultaneously.

With such a filter, I wouldn't have to play with individual branches filters.

**Comment by Otiel** [comment]
I second the original request, but as a senior developer. 😄

I'd like to be able to filter all local branches because in my use case they are often related and I'm working on them simultaneously.

With such a filter, I wouldn't have to play with individual branches filters.

**Comment by DanPristupov** [comment]
Filtering local branches is easy, just hide the remote.

<img width="311" alt="Screenshot 2023-11-16 at 12 45 50" src="https://github.com/fork-dev/TrackerWin/assets/618115/5ab200db-6e65-46b3-8d34-7dbb650ad147">

**Comment by narf** [comment]
> Filtering local branches is easy, just hide the remote.
> <img alt="Screenshot 2023-11-16 at 12 45 50" width="311" src="https://user-images.githubusercontent.com/618115/283437931-5ab200db-6e65-46b3-8d34-7dbb650ad147.png">

True, but in this case you can't see if the local branches are up to date with their remote tracked branch, or have been rebased or someone else might have added a commit or whatever. If there is a divergence it's a good thing to be able to see it right away.
